### PR TITLE
cleaned practice & rituals

### PR DESCRIPTION
**Pull request by dsugrobov**
* Moved "Contribution Conventions" to a technical value
* Removed Scrum attributes from a people value
* Added Design System
* Fixed spelling

### CODE REVIEW

File: content/en/values/people.md [hunk 1]
```diff
@@ -26,19 +26,13 @@ is a source of organizational and professional help and
...
-
 {{< glossary title="Retrospective" source="https://www.scrumguides.org/scrum-guide.html#events-retro" >}}
 The Retro is the event giving an opportunity for the team to inspect itself and create a plan for improvements to be enacted next time.
 {{< /glossary >}}
-
 {{< glossary title="InnerSource" >}}
 [Being InnerSource Ready](https://adeo.github.io/innersource) means that you are ready to collaborate with other developers, including developers in your team.
 {{< /glossary >}}
 
-{{< glossary title="Contribution conventions" source="https://adeo.github.io/innersource" >}}
```
Comment (vch-lm) [inline content/en/values/people.md:38]: I'm a bit sceptical about removing "Contribution conventions". I think they support sharing common culture quite well, and this practice should be present both here and in technologies.

File: content/ru/values/technology.md [hunk 2]
```diff
@@ -72,6 +75,10 @@ Infrastructure as a Code — Инфраструктура как Код. Мы п
 
 * Все бизнес настройки конфигурируемы через веб-интерфейс.
 
+{{< glossary title="Дизайн Система" source="https://vc.ru/flood/36947-chto-takoe-dizayn-sistemy-i-gde-oni-primenyayutsya">}}
```
Comment (vch-lm) [inline content/ru/values/technology.md:78]: What do you think about the same source for all languages?

Comment (dsugrobov) [reply content/ru/values/technology.md:78]: I think about the practical value of this: two different sources with additional information in each language I valued higher than the similarity between them.

File: content/en/values/technology.md [hunk 3]
```diff
@@ -72,6 +75,10 @@ Meetings are held at least once a month. Leader — CTO.
 
 * All business settings configurable through UI.
 
+{{< glossary title="Design System" source="https://uxdesign.cc/what-the-heck-is-a-design-system-c89a8ea73b0d">}}
+Design System – the system of rules and tools for visual and technical design. Reflects the philosophy of the product and constantly developing.
```
Comment (aleksandrregner) [inline content/en/values/technology.md:79]: who constantly developing? Design system or the philosophy?

Comment (aleksandrregner) [reply content/en/values/technology.md:79]: I think "Design System – the system of rules and tools for visual and technical design. Reflects the philosophy of the product. Design System continuously evolving." sounds a little bit better :)

**Comment by dsugrobov** [comment]
@vch-lm returned "contribution conventions" to the people's value.

**Pull request merged**